client/my-sites/ads/main.jsx

```diff
@@ -133,7 +133,7 @@ const AdsMain = React.createClass( {
 		if ( this.props.requestingWordAdsApproval || this.props.wordAdsSuccess ) {
 			notice = (
 				<Notice status="is-success" showDismiss={ false }>
-					{ this.translate( 'You have successfully joined WordAds. Please review these settings:' ) }
+					{ this.translate( 'You have joined the WordAds program. Please review these settings:' ) }
```

Comment:
It should be fine to keep the `successfully` here. eg `You have successfully joined the WordAds program! Please review these settings:`